sample/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">Auth0 SDK Sample</string>
+    <string name="com_auth0_domain">lbalmaceda.auth0.com</string>
+    <string name="com_auth0_client_id">PX2vJhpALUNT66NHdCdD30XWqlIR4oEZ</string>
+    <string name="com_auth0_scheme">demo</string>
```

Comment:
Should these be included in the diff?

Reply:
Yes, no problem. Have already been there for a while. It's a native app I have set up with the right callbacks for a quick download & test.